### ISSUE TITLE
Loader icon 404 when deploying at non-root path

### ISSUE THREAD

**Issue by naegelyd** [chop-dbhi/varify#196]
When varify is deployed at a nested level, such as /varify/ on a multi-app server, the loading indicator won't be shown on the results table. You will get the following 404 instead:

```
GET https://rescows02.research.chop.edu/static/images/loader-tiny.gif 404 (Not Found) 
```

**Comment by naegelyd** [comment]
This is unrelated to the overall problem of the results loverlay just not working altogether as outlined in cbmi/cilantro#500.

**Comment by naegelyd** [comment]
Fixed in 9577d47b6ae6d48e6cf2803c8
